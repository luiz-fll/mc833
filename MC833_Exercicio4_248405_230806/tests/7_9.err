read: Connection reset by peer
