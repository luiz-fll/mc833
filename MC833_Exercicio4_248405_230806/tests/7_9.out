local: 127.0.0.1:55336
remoto: 127.0.0.1:60007
